app/code/Magento/Reports/Controller/Adminhtml/Report/Shopcart/Customer.php

```diff
@@ -4,10 +4,22 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
+
 namespace Magento\Reports\Controller\Adminhtml\Report\Shopcart;
 
-class Customer extends \Magento\Reports\Controller\Adminhtml\Report\Shopcart
+use Magento\Framework\App\Action\HttpGetActionInterface as HttpGetActionInterface;
+
+/**
+ * Class \Magento\Reports\Controller\Adminhtml\Report\Shopcart\Customer
+ */
+class Customer extends \Magento\Reports\Controller\Adminhtml\Report\Shopcart implements HttpGetActionInterface
 {
+    /**
+     * Authorization of a customer report
+     */
+    const ADMIN_RESOURCE = null;
```

Comment:
Setting `ADMIN_RESOURCE ` to null does not look right to me. There should be an admin resource defined for every controller.

Reply:
@sivaschenko Yes, but in the current source code, it is null. So is it ok if I change it to other value?

Reply:
As I see, this controller isn't used in anywhere, and no ACL for it. So I will remove it and it will have the admin resource of the parent class.